### DISCUSSION TITLE
Python debuggers for PyScript

### DISCUSSION THREAD

**Discussion by KennyAWT** [pyscript/pyscript#1913] [Proposals]
Hi , I would like that you can  implement a debugger how I can see in brython , or an extension for a text editor or ide . I hope it can be possible soon.

Thank you!

**Comment by WebReflection** [comment]
The debugger is strongly interpreter dependent and I am afraid I am not sure what's the current state in Pyodide or MicroPython but it's nothing we can implement by our own IIRC and AFAIK. There will be progress around this issue but we need to be aligned with interpreters too, thanks for your understanding.
